status-computation/java/src/main/java/ar/PickEndpoints.java

```diff
@@ -142,6 +160,17 @@ public Boolean exec(Tuple input) {
 
 		// If filtering by profiles is enabled
 		if (this.filter == 1) {
+			
+			// Filter By monitoring engine 
+			try {
```

Comment:
@kaggis it is not clear to me why this `try` clause is included within the filter section. Maybe it should it be moved outside? What do you think?